lact-gui/src/app/info_row.rs

```diff
@@ -131,6 +131,7 @@ mod imp {
                         set_halign: gtk::Align::End,
                         set_use_markup: true,
                         set_ellipsize: pango::EllipsizeMode::End,
+                        add_css_class: "monospace",
```

Comment:
Please add this only for dynamic values (such as `gpu_stats_section`), it should not affect things like the hardware info page.